aleph2:
  joint_limits:
    gimbal1_pan:
      has_position_limits: false
      has_velocity_limits: true
      max_velocity: 2.0
      has_acceleration_limits: false
      has_jerk_limits: false
      has_effort_limits: false